pages/api/v1/goalies/ratings/index.ts

```diff
@@ -2,8 +2,8 @@ import Cors from 'cors';
 import { NextApiRequest, NextApiResponse } from 'next';
 import SQL from 'sql-template-strings';
 
-import { query } from '../../../../lib/db';
-import use from '../../../../lib/middleware';
+import { query } from '../../../../../lib/db';
```

Comment:
moved original stats and ratings files into a folder and changed names to "index.ts".  This allows for the use of [id] in all cases.